autofit/graphical/declarative/result.py

```diff
@@ -20,7 +19,7 @@ def __init__(
         ----------
         ep_history
             A history of the optimisation
-        declarative_factor
+        declarative_factor: AbstractDeclarativeFactor
```

Comment:
Type in _iinit_ and not docstring?

Reply:
it's because it creates an import recursion